Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityControllerMappingProfile.asset

```diff
@@ -2870,6 +2870,19 @@ MonoBehaviour:
       axisCodeY: AXIS_18
       invertXAxis: 0
       invertYAxis: 0
+    - id: 11
```

Comment:
We should probably make sure the ID, description, and any other stale data from the create button (axisCodeX/Y as well) are reset for this action.

Might be an unrelated bug in the generic action mapper that everything is copied from the previous action, even if you aren't able to change it in the UI.

Reply:
Perfect, thanks!

---
In reply to: [286192378](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/4484#discussion_r286192378) [](ancestors = 286192378)